OrderService/OrderService.Domain/Abstractions/Data/ITemporaryStorageService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using OrderService.Domain.Entities;
+
+namespace OrderService.Domain.Abstractions.Data;
+
+public interface ITemporaryStorageService
+{
+    public void SaveCart(Dictionary<int, ProductEntity> cart);
```

Comment:
what is used as a key in the dictionary? why it's int?

Reply:
Key is a product id, that's why is's int